testing/framework/TestCommonTests.py

```diff
@@ -1975,43 +2001,47 @@ def raise_exception(*args, **kw):
         """)
 
         expect_stderr = lstrip(
-            fr"""Exception trying to execute: \[{re.escape(repr(sys.executable))}, '[^']*pass'\]
+            rf"""Exception trying to execute: \[{re.escape(repr(sys.executable))}, '[^']*pass'\]
 Traceback \(most recent call last\):
   File "<stdin>", line \d+, in (\?|<module>)
   File "[^"]+TestCommon\.py", line \d+, in run
     super\(\)\.run\(\*\*kw\)
   File "[^"]+TestCmd\.py", line \d+, in run
-    p = self\.start\(program=program,
+    p = self\.start\(
 (?:\s*\^*\s)?  File \"[^\"]+TestCommon\.py\", line \d+, in start
     raise e
   File "[^"]+TestCommon\.py", line \d+, in start
-    return super\(\)\.start\(program, interpreter, arguments,
+    return super\(\)\.start\(
 (?:\s*\^*\s)?  File \"<stdin>\", line \d+, in raise_exception
 TypeError: forced TypeError
-""")
+"""
+        )
 
         # Python 3.13+ expanded error msgs again, not in a way we can easily
         # accomodate with the other regex.
+        # TODO: broken again after reformat work
         expect_enhanced_stderr = lstrip(
-            fr"""Exception trying to execute: \[{re.escape(repr(sys.executable))}, '[^']*pass'\]
+            rf"""Exception trying to execute: \[{re.escape(repr(sys.executable))}, '[^']*pass'\]
 Traceback \(most recent call last\):
   File "<stdin>", line \d+, in (\?|<module>)
   File "[^"]+TestCommon\.py", line \d+, in run
     super\(\)\.run\(\*\*kw\)
 (?:\s*[~\^]*\s*)?File "[^"]+TestCmd\.py", line \d+, in run
-    p = self\.start\(program=program,
-                   interpreter=interpreter,
-    \.\.\.<2 lines>\.\.\.
-                   timeout=timeout,
-                   stdin=stdin\)
+    p = self\.start\(
+        program=program,
```

Comment:
Is this a black induced change?

Reply:
It has to match what is actually being produced, and *that* was a black-induced change.  There are some changes to common and cmd tests that are for that reason, rather than specifically themselves having been made by black.

Reply:
Gotcha.